docs/reference/search/index.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Search
+parent: Objects
```

Comment:
Can we add this to the tags section instead of objects, this should be reserved for drops really